test/Microsoft.AspNetCore.Razor.Evolution.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/AddTagHelperDirective_DesignTime.mappings.txt

```diff
@@ -0,0 +1,10 @@
+Source Location: (14:0,14 [17] TestFiles/IntegrationTests/CodeGenerationIntegrationTest/AddTagHelperDirective.cshtml)
+|"*, TestAssembly"|
+Generated Location: (423:10,30 [17] )
+|*, TestAssembly";|
```

Comment:
Still wrong.

Should also add a non-quoted case to the AddTagHelperDirective.cshtml file.